bundles/org.openhab.binding.mikrotik/README.md

```diff
@@ -0,0 +1,374 @@
+# Mikrotik RouterOS Binding
+
+This binding integrates [Mikrotik](https://mikrotik.com/) [RouterOS](https://help.mikrotik.com/docs/display/ROS/RouterOS)
+[devices](https://mikrotik.com/products) allowing monitoring of system resources, network interfaces and WiFi clients.
+
+## Supported Things
+
+* `routeros` - An instance of the RouterOS device connection
+* `interface` - A network interface inside RouterOS device
+* `wifiRegistration` - Any wireless client connected to a RouterOS wireless network (regular or CAPsMAN-managed)
+
+
+## Discovery
+
+Discovery is currently not supported, but may be implemented in future versions.
+
+
+## Binding Configuration
+
+The binding has no configuration options, all configuration is done at the Bridge and Thing levels.
+
+
+## Bridge Configuration
+
+To use this binding you need at least one RouterOS-powered device (Bridge) accessible to the host running
+openHAB via network.
+Make sure your RouterOS has the API enabled by visiting [<kbd>IP -> Services</kbd>](https://wiki.mikrotik.com/wiki/Manual:IP/Services) 
+configuration section in 
+[WinBox](https://wiki.mikrotik.com/wiki/Manual:Winbox).
+Take note of the API port number as you'll need it below.
+[SSL API connection](https://wiki.mikrotik.com/wiki/Manual:API-SSL) is not yet supported by this binding.
+To connect to the RouterOS API, you will need to provide user credentials for the bridge thing. 
+You may use your current credentials that you use to manage your devices, but it is highly recommended to **create a read-only RouterOS user** since this binding only need to read data from the device.
+To do this, proceed to <kbd>System -> Users</kbd> configuration section and add a user to the `read` group.
+
+> Thing type: `routeros`
+
+The RouterOS Bridge configuration parameters are:
+
+* `host` _(required text)_ [Default: `192.168.88.1`] - Hostname or IP address of the RouterOS device
+* `port` _(optional integer)_ [Default: `8729`] - API Port number of the RouterOS device
+* `login` _(required text)_ [Default: `admin`] - The user login to access the the RouterOS device
+* `password` _(required text)_ - The user password to access the RouterOS device
+* `refresh` _(optional integer)_ [Default: `10`] - The refresh interval in seconds to poll the RouterOS device
```

Comment:
You could make this a table to be consistent with the other bindings.

Reply:
Is there a tool to build Markdown table out of XML? That's how this description is generated - via a script.

Reply:
Yes, @cpmeister wrote a Java class for generating the markdown table from the XML files. But I'm afraid it's burried in some old PR.

Reply:
Can we leave it like that for now?
I remember some bindings has parameter descriptions as lists, so I guess that's acceptable. Anyway, I'd be happy to reformat with tables once the tool will be available, or when I'll make my own (I was thinking about making browser JS app one day).

Reply:
There are several old addons which don't use tables. New addond should have tables to achieve consistentency across the manifold addon readmes. A browser-based converter would be pretty cool!